qiskit/circuit/classicalfunction/boolean_expression.py

```diff
@@ -25,15 +25,17 @@
 class BooleanExpression(ClassicalElement):
     """The Boolean Expression gate."""
 
-    def __init__(self, expression: str, name: str = None) -> None:
+    def __init__(self, expression: str, name: str = None, var_order: list = None) -> None:
         """
         Args:
             expression (str): The logical expression string.
-            name (str): Optional. Instruction gate name. Otherwise part of
-                        the expression is going to be used.
+            var_order(list): A list with the order in which variables will be created.
+               (default: by appearance)
+            name (str): Optional. Instruction gate name. Otherwise part of the expression is
+               going to be used.
```

Comment:
```suggestion
            name (str): Optional. Instruction gate name. Otherwise part of the expression is
               going to be used.
            var_order(list): A list with the order in which variables will be created.
               (default: by appearance)
```